src/index.js

```diff
@@ -1 +1,93 @@
+/* eslint-disable linebreak-style */
 import './style.css';
+
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/oaP1XqMFPoB60pz4ScpD/scores/';
+
+const submitBtn = document.getElementById('submitBtn');
+
+// const constructJson = (name, score) => {
+//   score = Number(score);
+//   const user = {
+//     user: name,
+//     score,
+//   };
+//   return user;
+// };
+
+const displayResult = (user) => {
+  const ul = document.getElementById('ulList');
+
+  user.forEach((player) => {
+    const li = document.createElement('li');
+    li.classList.add('table-row');
+
+    const nameText = document.createElement('p');
+    nameText.innerHTML = `${player.user}: ${player.score}`;
+
+    li.appendChild(nameText);
+    ul.appendChild(li);
+  });
+};
+
+const request = new XMLHttpRequest();
+async function getUser(url) {
```

Comment:
- [ ] You are encouraged to make use of ES6 syntax which makes the code look neater and more readable by using arrow functions.

Reply:
Refer to the comment below (it's about the same function in your two comments)

> I agree that the Fetch method is more reliable, but considering that the code is already efficient as it is, and the fetch method is not the only way and is also not a mandatory requirement in Micrvoverse guidelines, in my opinion, this request to change the code should be only OPTIONAL because is not about an inefficient code but only about preferences. Furthermore, switching now to the fetch method would also require additional changes in the whole js structure, leading only to a delayed approval and not necessarily to a better code.